src/utils/hints/namesHints.js

```diff
@@ -23,4 +23,8 @@ export const namesHints = {
   eventTransactionHash: 'Transaction hash of the Name event.',
   eventTime: 'Block height and estimated date and time when the event was triggered.',
   activity: 'Type of event that was triggered.',
+  recentlyActivatedData: 'The Name and its owner who recently activated it. Depending on the length a Name is activated either through an auction or by direct acquisition.',
```

Comment:
Here I would explain precisely how it works (13 characters threshold). Personally I would use some simpler word than acquisition